spec/spec_helper.rb

```diff
@@ -63,4 +63,6 @@
     #puts "travelling to #{now}"
     Timecop.travel(now)
   end
+
+  config.before(:each) { ActionMailer::Base.deliveries.clear }
```

Comment:
I don't think we want to do this before *every* test. I think we should just do it on the ones that need it. I tend to use `expect { do_something }.to change(ActionMailer::Base.deliveries, :count).by(1).